### ISSUE TITLE
How to maximize neuron coverage?

### ISSUE THREAD

**Issue by BT123** [peikexin9/deepxplore#11]
Hi~@peikexin9
In my experiment, I use the parameters you provided and I also try 50 grad_iterations.
The neuron which is random selected in the program, will not be activated after all iterations done.
So, I want to find the answer about how to increase the neuron coverage of one model, but I still didn't understand.
If the grad_iterations is not big enough to find an input to activate the neuron, when the program find the differential-inducing input?
Thank you~

**Comment by peikexin9** [comment]
Hi @BT123 you can also try to set larger learning rate in gradient descent, or increase the weight of prioritizing increasing neuron coverage rather than prioritizing finding difference inducing input. Also there may exist some "dead" neurons that are extremely hard or can never to be activated.
